### ISSUE TITLE
[Feature] 3D product in an environment

### ISSUE THREAD

**Issue by ShwetKhatri2001** [ShwetKhatri2001/AR-Webstore#114]
### Description

Instead of showing the 3D product / model in the plain gray area, Why not show it in an environment where it's actually going to be placed after the purchase !

Take a look at the small changes made in this [model-viewer example](https://modelviewer.dev/examples/lightingandenv/#hdrSkyboxImage) and the screenshot below to add a background and lighting using `skybox` ,`environment-image` and `exposure`.

![image](https://github.com/ShwetKhatri2001/AR-Webstore/assets/56475750/3f483f1b-0cf6-48b2-bde6-95e2f5c3118b)

Similarly, create a dropdown having at least three backgrounds suitable for the product. 
( For example, if it's a gym product then provide different gym environments.

Keep the one background by default in which the product looks the best.
Also, provide an option `None` to see the product without any background in that plain-gray area.

### Screenshots

It's just going to be a dropdown for now between the viewer-gray-area and QR-Code.
When a background from the dropdown is selected it should be applied immediately. 

Don't hardcode the background-image sources and keep the list of backgrounds separate in an object. 
Because there will be different list of background depending on the product and the changes you make shoould be adaptable to it.

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines]
- [ ] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

**Comment by ShwetKhatri2001** [comment]
Go ahead. This might be a good starting point for you to learn about [model-viewer](https://modelviewer.dev/) since you've also shown interest to work on other related issues. So, implement this and explore.

**Comment by mandliyarajendra11** [comment]
@ShwetKhatri2001 not done , skybox is not good option in chair product , not looking good and making background blur

**Comment by ShwetKhatri2001** [comment]
Yes, it might look blur if the image size is bigger than the container, but for full-screen view it should work. 
You can find some hdr background images online which might fit for such case.

As I said, It's just about providing an option to view the product in three different backgrounds and if none of the backgrounds look good then the current plain-gray background it has will be shown by default.

**Comment by ShwetKhatri2001** [comment]
Any updates or issues @mandliyarajendra11 ?

**Comment by ShwetKhatri2001** [comment]
Any updates or issues @mandliyarajendra11  ?
Please try to make your changes and raise your PR within the next 2 days.
After that, This issue will be open to other interested contributors.
